vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -3338,18 +3328,19 @@ EXPORT(uint32_t, sceGxmProgramParameterGetContainerIndex, const SceGxmProgramPar
 
 EXPORT(uint32_t, sceGxmProgramParameterGetIndex, const SceGxmProgram *program, const SceGxmProgramParameter *parameter) {
     TRACY_FUNC(sceGxmProgramParameterGetIndex, program, parameter);
-    uint64_t parameter_offset = program->parameters_offset;
-
-    if (parameter_offset > 0)
-        parameter_offset += (uint64_t)&program->parameters_offset;
-    return (uint32_t)((uint64_t)parameter - parameter_offset) >> 4;
+    auto parameters = program->program_parameters();
+    if (parameters) {
+        return (uint32_t)((parameter - parameters) / sizeof(SceGxmProgramParameter));
```

Comment:
```suggestion
        return static_cast<uint32_t>(parameter - parameters);
```
Doing (T*)a - (T*)b return ((size_t)a - (size_t)b) / sizeof(T)